cmd/snap-bootstrap/triggerwatch/triggerwatch.go

```diff
@@ -31,6 +31,10 @@ import (
 	"github.com/snapcore/snapd/osutil/udev/netlink"
 )
 
+var (
+	timeAfter = time.After
```

Comment:
am I confused? this doesn't seem to be used in the diff

Reply:
I might have broken it.

Reply:
Yes. That was because I went back and forth with the path for the fix, and then I forgot to change it back to what was correct before pushing. It should be fixed now.